CONTRIBUTION.md

```diff
@@ -3,92 +3,106 @@
 subject line should feature the what and the body of the commit should describe the why.
 
-The format can be described more formally as follows:
-
+A good commit message should look like 
 ```
 [subsystem] <what changed>
-<BLANK LINE>
+
 <why this change was made>
-<BLANK LINE>
-<footer>
+
+<Footer>
 ```
-Example for a sample feature:
+
+A real world example would look like 
 ```
 [docs] Add the Guidlines
 
-Cupcake ipsum dolor sit. Amet tart cheesecake tiramisu chocolate cake topping.
-Icing ice cream sweet roll. Biscuit dragée toffee wypas.
+adds the contribution guidelines for maintaining consistent style across the repo 
 
 Follow-up to Id5e7cbb1.
 ```
-
-The first line is the subject and should be no longer than 50 characters, the second line is always blank, and other
-lines should be wrapped at 80 characters. This allows the message to be easier to read on GitHub as well as in various
-git tools.
+The subject should be no longer than 50 characters, and the body should be no longer than 80 characters. There are some [githooks](https://github.com/jorisroovers/gitlint) you can install that can help enforce these style requirements.
 
 If it is a bug fix commit, the bug identifier should be mentioned in the commit message as `Fixes OCPBUGS-123` in a separate
 line.
 
-### PR workflow
-
-- Before submitting a PR
-  - Format your code and fix all the spelling/grammatical mistakes.
-  - Limit the column length of the code and the comments within the code to 120 characters.
-  - Error messages within the code should be limited to a single line.
-  - Update the documentation if your PR is introducing or changing user facing functionality.
-  - Ensure each PR commit compiles and all unit and e2e tests pass on your local machine.
-    - PRs that have vendor commits are an exception to this rule.
-- Following are the things you should keep in mind once you open a PR:
-  - Add a hold as soon as you open the PR by commenting `/hold`
-  - Wait for comments from at least 2 reviewers before pushing changes.
-  Open comments in the meanwhile can be worked on locally.
-  - PR comments should be addressed in new commits. Before final approval, they have to be squashed.
-  - If PR has multiple commits, changes requested should eventually be squashed into the original commit where the
-  change was requested before cancelling the hold. Each commit in the final PR before merge should pass the tests and be
-  usable. (see below)
-  - Once the PR is approved, remove the hold by commenting `/hold cancel` to merge the PR
+## Opening a pull request 
+Once you are done with the pre-PR checklist, push your changes and create a pull request! 
+
+### Format of the pull request 
+#### Title 
+When creating your pull request, you should prefix it with the jira issue, followed by the subsystem name in brackets. It should follow the format
+
+```WINC-<number>: [<subsystem>] <title>```
```

Comment:
What about bug fixes?

Reply:
This has not been addressed.

Reply:
https://github.com/openshift/windows-machine-config-operator/pull/1481/files#diff-0c8a93902edcf8795d61d5e98dc43d4ecd8645f4a924e67c080a70f364edc78eR87 

should it be more direct than it being here ? 
It makes sense to put all of the bugfix information in one place.

Reply:
`WINC/OCPBUGS-<number>` will suffice here.